libkbfs/favorites.go

```diff
@@ -134,18 +139,27 @@ type favoritesCacheEncryptedForDisk struct {
 }
 
 func (f *Favorites) readCacheFromDisk(ctx context.Context) error {
+	log := f.config.MakeLogger("Favorites")
```

Comment:
You call this multiple times. It might be better to store a logger instead. `f.config.MakeLogger("FAV")` (we use 3 letter codes to keep the log prefix line length the same).